tests/test_hero_and_monster.py

```diff
@@ -0,0 +1,18 @@
+from gamebook.hero import Hero
+from gamebook.monster import Monster
+from gamebook.trait import Trait
+
+
+def test_fight():
+    skill = Trait("Skill", 6, 1)
```

Comment:
Try to use the optional notation (`funcname(paramname = somevalue)` instead of `funcname(somevalue)`) because this will be kind of a documentation for your code.